test.sh

```diff
@@ -9,6 +9,18 @@ if [ $? -ne 0 ]; then
     echo "**ERROR**: failed to initialize Python Virtual Environment" && exit 1
 fi
 
+# Rename all project manifests to correct format.
+# This is done to avoid security scanning them for deprecated/vulnerable library dependancies
+# These sample projects are not intended to be used in production, but are used for testing
+mv ./sample-projects/dotnet-samples/sample_csproj_PLACEHOLDER ./sample-projects/dotnet-samples/sample.csproj
+mv ./sample-projects/go-samples/compatible/go_mod_PLACEHOLDER ./sample-projects/go-samples/compatible/go.mod
+mv ./sample-projects/go-samples/incompatible/go_mod_PLACEHOLDER ./sample-projects/go-samples/incompatible/go.mod
+mv ./sample-projects/java-samples/pom_xml_PLACEHOLDER ./sample-projects/java-samples/pom.xml
+mv ./sample-projects/node-samples/package_json_PLACEHOLDER ./sample-projects/node-samples/package.json
+mv ./sample-projects/python-samples/compatible/requirements_txt_PLACEHOLDER ./sample-projects/python-samples/compatible/requirements.txt
+mv ./sample-projects/python-samples/incompatible/requirements_txt_PLACEHOLDER ./sample-projects/python-samples/incompatible/requirements.txt
+mv ./sample-projects/ruby-samples/Gemfile_PLACEHOLDER ./sample-projects/ruby-samples/Gemfile
+
```

Comment:
Same here

Reply:
Addressed -  https://github.com/aws/porting-advisor-for-graviton/pull/54#discussion_r1593280800